src/modules/rules.ts

```diff
@@ -263,7 +263,7 @@ export const ruleCustomDataHandlers: {
 				const e = page * PAGE_SIZE + i;
 				if (e >= value.length)
 					break;
-				MainCanvas.strokeRect(1050, Y + 26 + i * 70, 766, 64);
+				DrawRect(1050, Y + 26 + i * 70, 766, 64, "black");
```

Comment:
This is a stroke, not fill:
```suggestion
				DrawEmptyRect(1050, Y + 26 + i * 70, 766, 64, "black");
```